chrono-tz/src/directory.rs.example

```diff
@@ -1,3 +1,5 @@
+pub const IANA_TZDB_VERSION : &str = "2023c";
```

Comment:
Why do we have this?

Reply:
This file is showing a subset of the generated source; I decided to include the version since the actual `directory.rs` will have it like this.